src/config.rs

```diff
@@ -39,6 +39,7 @@ pub(crate) struct Config {
     // We want this validation to run even without the entry in the config file
     #[serde(default = "ValidateConfig::default")]
     pub(crate) validate_config: Option<ValidateConfig>,
+    pub(crate) pr_tracking: Option<ReviewPrefsConfig>,
```

Comment:
What is this for?

Reply:
I wish there was documentation for that!

IIUC it's used to ensure that the `Config` struct maps correctly the content of the `triagebot.toml`. In other words, when we will add `[pr-tracking]` in the `triagebot.toml`, it must be also mapped here (will be used by the `issue_handler!()` macro).

Reply:
Hmm, it's pretty unfortunate it's so hard-coded as an issue handler. I guess it makes sense because we only want to get webhooks for the rust-lang/rust repo, but we can't really support other repos anyways with the current setup.

Reply:
Can you please add a comment about this?

Reply:
ok, I'll add some wording about that. However the issue handler (well, any handler in that macro) can be enabled by adding a table thing `[something]` in the .toml files. Without that handlers will be just ignored when a webhook payload comes.

(iiuc what you mean)

Supporting other repositories (I assume you mean for PR tracking), I think it should be relatively easy and involve small changes at the DB table level (like adding a field).

Reply:
added some wording. hope it clarifies :slightly_smiling_face: